/**/*:
  ros__parameters:
    storage:
      url: "http://127.0.0.1:8383"
      api_token: "access_token"
      bucket: "ros_data"
    pipelines:
      test_data:
        filename_mode: "incremental"
        include_topics:
          - "/nissan/.*"
        exclude_topics:
        static_labels:
          source: test_data
          robot: alpha
        split:
          max_duration_s: 10
          max_size_bytes: "5MB"
      rosout:
        filename_mode: "incremental"
        include_topics:
          - "/rosout"
        split:
          max_duration_s: 10
          max_size_bytes: "5MB"